server/gc/src/test/java/org/apache/accumulo/gc/GarbageCollectWriteAheadLogsTest.java

```diff
@@ -234,4 +257,288 @@ public void testIsUUID() {
     assertFalse(GarbageCollectWriteAheadLogs.isUUID("0" + UUID.randomUUID().toString()));
     assertFalse(GarbageCollectWriteAheadLogs.isUUID(null));
   }
+
+  @Test
+  public void testTimeToDeleteTrue() throws InterruptedException {
+    gcwal.clearFirstSeenDead();
+    HostAndPort address = HostAndPort.fromString("tserver1:9998");
+    long wait = AccumuloConfiguration.getTimeInMillis("1s");
+    gcwal.timeToDelete(address, wait); // to store first seen dead
+    sleep(wait * 2);
+    assertTrue(gcwal.timeToDelete(address, wait));
+  }
+
+  @Test
+  public void testTimeToDeleteFalse() {
+    HostAndPort address = HostAndPort.fromString("tserver1:9998");
+    long wait = AccumuloConfiguration.getTimeInMillis("1h");
+    long t1, t2;
+    boolean ttd;
+    do {
+      t1 = System.nanoTime();
+      gcwal.clearFirstSeenDead();
+      gcwal.timeToDelete(address, wait);
+      ttd = gcwal.timeToDelete(address, wait);
+      t2 = System.nanoTime();
+    } while (t2 - t1 > (wait / 2) * 1000000); // as long as it took less than half of the configured wait
```

Comment:
I saw you used TimeUnit earlier in the patch to convert time.  Could use TimeUnit earlier in this test method to convert `wait` to nanos.  IMO would be slightly more readable than multiplying by 1000000.